miniconda/Dockerfile

```diff
@@ -1,10 +1,12 @@
-FROM debian:7.4
+FROM ubuntu:14.04
 
 MAINTAINER Travis Swicegood
 
-RUN apt-get update && apt-get install -y wget bzip2
+RUN apt-get update
+RUN apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/anaconda/bin:$PATH' > /etc/profile.d/conda.sh
-RUN wget --quiet http://repo.continuum.io/miniconda/Miniconda3-3.6.0-Linux-x86_64.sh && \
-    /bin/bash /Miniconda3-3.6.0-Linux-x86_64.sh -b -p /opt/anaconda && \
-    rm Miniconda3-3.6.0-Linux-x86_64.sh
```

Comment:
I didn't know that every use of RUN behaves in that way.
Thanks. I adjusted the code.